tests/small/LandIce/SHMIP/input_bueler_steady.yaml

```diff
@@ -0,0 +1,219 @@
+%YAML 1.1
+---
+ANONYMOUS:
+  Build Type: Tpetra
+  Debug Output: 
+    Write Jacobian to MatrixMarket: 0
+    Write Solution to MatrixMarket: 0
+  Problem: 
+    Phalanx Graph Visualization Detail: 0
+    Solution Method: Continuation
+    Name: LandIce Hydrology 2D
+    Response Functions: 
+      Number Of Responses: 0
+      Response 0: 
+        Type: Scalar Response
+        Name: Squared L2 Difference Source ST Target PST
+        Field Rank: Scalar
+        Source Field Name: water_thickness
+        Target Value: 0.0
+    Initial Condition: 
+      Function: Constant
+      Function Data: [100.0, 0.1] # [ p_w (in kPa), h (in m) ]
+    Dirichlet BCs: 
+      DBC on NS BoundaryNodeSet1 for DOF water_pressure: 0.0
+    Neumann BCs: { }
+    Parameters: 
+      Number Of Parameters: 1
+      Parameter 0: 
+        Type: Scalar
+        Name: Homotopy Parameter
+        Nominal Value: 0.0
+    LandIce Physical Parameters: 
+      Water Density:        1000.0        # [ kg m^-3 ]
+      Ice Density:          910.0         # [ kg m^-3 ]
+      Ice Softness:         3.1689e-15    # [ kPa^-3 s^-1 ]
```

Comment:
Ha! This is the flow factor, right?  I propose to have this in SI units [Pa^{-3} s^{-1}] and have the Stokes viscosity use this, properly scaled, when the flow factor is uniform. Do you gree?

Reply:
Yes, it's the flow factor. In the hydrology papers is usually referred to as Ice Softness. I guess we should uniform the notation on that too.

I'm ok switching to SI units. I used kPa cause that's the pressure units we use in ALI, but SI units everywhere is probably the best way, so that users don't have to wonder about units when setting values in input files.

Reply:
OK, I can change the Viscosity at a later time, to link at this parameter.